tap_linkedin_ads/client.py

```diff
@@ -114,13 +116,28 @@ def raise_for_error(response):
 class LinkedinClient:
     def __init__(self,
                  access_token,
-                 user_agent=None):
+                 user_agent=None,
+                 timeout_from_config=None):
         self.__access_token = access_token
         self.__user_agent = user_agent
         self.__session = requests.Session()
         self.__base_url = None
         self.__verified = False
 
+        # if the 'timeout_from_config' value is 0, "0", "" or not passed then set default value of 300 seconds.
+        if timeout_from_config and float(timeout_from_config):
+            # update the request timeout for the requests
+            self.request_timeout = float(timeout_from_config)
+        else:
+            # set the default timeout of 300 seconds
+            self.request_timeout = REQUEST_TIMEOUT
+
+    # during 'Timeout' error there is also possibility of 'ConnectionError',
+    # hence added backoff for 'ConnectionError' too.
+    @backoff.on_exception(backoff.expo,
```

Comment:
@harshpatel4crest Add comment that why we have added backoff here instead of check_access_token()

Reply:
Added.